recipe/meta.yaml

```diff
@@ -7,7 +8,7 @@ package:
 source:
   fn: rasterio-{{ version }}.tar.gz
   url: https://github.com/mapbox/rasterio/archive/{{ version }}.tar.gz
-  sha256: 472908e335fc9a4d072fb6f41e08b58eb7120f18f6c12e28ffd3beb86edd7e1f
+  sha256: {{ sha256 }}
```

Comment:
Thanks! This makes a ton of sense and in hindsight the jinjaing really doesn't add much here... I've force pushed this away! 

Good to hear you've automated some of this! Can't imagine maintaining all of those feedstocks...